agasc/agasc.py

```diff
@@ -55,7 +59,8 @@ def read_ra_dec(self):
             # Now copy to separate ndarrays for memory efficiency
             return radecs['RA'].copy(), radecs['DEC'].copy()
 
-RA_DECS_CACHE = {DEFAULT_AGASC_FILE: RaDec(DEFAULT_AGASC_FILE)}
+if DEFAULT_AGASC_FILE is not None and os.path.exists(DEFAULT_AGASC_FILE):
```

Comment:
Is this somewhat reasonable?  Not sure if there is a more reasonable way to just defer setting up this cache until the user actually wants to use the agasc (not just import).